content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -144,7 +144,7 @@ tolerations:
   tolerationSeconds: 3600
 ```
 
-means that if this pod is running and a matching taint is added to the node, then
+means that if this pod is running without required toleration and taint is added to the node, then
```

Comment:
IMHO, this does not add any value and afaik the suggestion is technically incorrect. The example is trying to demonstrate the value of applying toleration to a pod with the boundary of time (3600 seconds) added. 

If the pod is running with the toleration & a matching taint is added to the node then the pod will stay bound to that node for 3600 seconds and then be evicted. Think of it as a grace period for which the pod can stay bound to the node modulo the condition that it remains tainted for that entire period.

That being said, I vote for the original text to be retained. 

```suggestion
means that if this pod is running and a matching taint is added to the node, then
```